File: TD/TD1/Code/4.cpp
/**
 * Demander un entier entre les bornes imposées
 */
#include <iostream>

int main()
{
  int a;

  std::cout << "Donne moi un entier entre 1 et 31 : " << std::flush;
  std::cin >> a;

  while (a < 1 || a > 31) {
    std::cout << "Donne moi un entier ENTRE 1 et 31 : " << std::flush;
    std::cin >> a;
  }

  std::cout << "Tout est OK !" << std::endl;

  return 0;
}

File: TD/TD1/Code/Python/4.py
#!/usr/bin/python
# Demander un entier entre les bornes imposées
print('Donne moi un entier entre 1 et 31 : ', end='')
a = int(input())

while a < 1 or a > 31:
    print('Il faut un entier entre 1 et 31 : ', end='')
    a = int(input())

print('Tout est OK !')
